passthrough/passthrough.go

```diff
@@ -247,18 +247,32 @@ type passthroughInlineTransformer struct {
 
 var PassthroughInlineTransformer = &passthroughInlineTransformer{}
 
+const PASSTHROUGH_MARKED_FOR_DELETION = "passthrough_marked_for_deletion"
+const PASSTHROUGH_PROCESSED = "passthrough_processed"
```

Comment:
This is Java/C styled constants -- and I assume they don't need to be be exported, so `markedForDeletion` etc. would be better.

As a general comment to the API; I think it would be better if we export only the funcs/types that the end user would potentially need, which would make the API easier to use and would also possibly make it easier to change names etc. later (without having to worry about breaking API).

Reply:
sorry, golang newbie here. The "capitalize to export" convention is a surprise :)

Will fix.